lightning/src/ln/channelmanager.rs

```diff
@@ -7082,6 +7082,16 @@ where
 			channel_id: Some(prev_hop.channel_id),
 		};
 
+		if prev_hop.counterparty_node_id.is_none() {
+			let payment_hash: PaymentHash = payment_preimage.into();
+			panic!(
+				"Prior to upgrading to LDK 0.1, all pending HTLCs must be resolved. It appears at least the HTLC with payment_hash {} (preimage {}) was not resolved. Please downgrade to LDK 0.0.124 and resolve the HTLC prior to upgrading.",
```

Comment:
Should we say 0.0.125 instead of 0.0.124? Also, shouldn't "all pending HTLCs must be resolved" be for those prior to 0.0.124?